stock_picking_auto_create_lot/tests/test_stock_picking_auto_create_lot.py

```diff
@@ -146,20 +145,19 @@ def test_multi_auto_create_lot(self):
         self.assertEqual(len(lot), 6)
 
     def test_immediate_validate_tracked_move_with_auto_create_lot(self):
-        picking_form = Form(self.env["stock.picking"])
-        picking_form.picking_type_id = self.picking_type_in
-        with picking_form.move_ids_without_package.new() as move:
-            move.product_id = self.product_serial
-            move.product_uom_qty = 4
-        receipt = picking_form.save()
-        receipt.action_confirm()
-
-        immediate_wizard = receipt.button_validate()
+        # Clear existing move if not the picking will open backorder wizard because
+        # when we manually assign lot for serial_not_auto product, other products still
+        # have 0 done qty.
```

Comment:
I think we do not need to be so verbose as the intention should be clear.
```suggestion
        # Clear existing moves.
```